RNScreens.podspec

```diff
@@ -19,6 +20,7 @@ min_supported_visionos_version = "1.0"
 
 rnscreens_cpp_flags = []
 
+rnscreens_cpp_flags << "-DREACT_NATIVE_MINOR_VERSION=#{rnscreens_find_react_native_minor_version()}"
```

Comment:
You don't need this. There is `ReactNativeVersion.h` header which defines for you. I think we can use it in gamma project.

Reply:
https://github.com/software-mansion/react-native-screens/pull/3274/commits/bef37e0a22a3004ced9e8d8843ab3e9f75b26ddd